washingtonpost/klaxon
# NAPPS-1348: fix logo styling across browsers

While it appears that the Trident URL redirects are making it such that the Post logo doesn't always load, I think the bigger problem is that the display is wonky on other browsers. Hoping that this styling is better and makes the changes more easily contributed back to the original Klaxon repo. 

Once this is merged in and seems to be working, I'll play around with the issue of the logo only sometimes loading.

[NAPPS-1348]

Testing steps:
- run app locally in both Chrome, Firefox and Safari and note the WP logo looks similar
- Log into the app and note that while browsing, the logos appear at the top left



[NAPPS-1348]: https://arcpublishing.atlassian.net/browse/NAPPS-1348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
